src/cascadia/TerminalApp/App.cpp

```diff
@@ -270,6 +273,43 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Flyout(newTabFlyout);
     }
 
+    // Method Description:
+    // - Builds the taskbar jumplist and populates it with the user's profiles. Clicking
+    //   the jumplist items will open a new tab associated with the clicked profile.
+    // Arguments:
+    // - None
+    // Return Value:
+    // - None
+    IAsyncAction App::_CreateJumplist()

```

Comment:
this can probably be a `winrt::fire_and_forget`.. i think?

Reply:
I was having some issue getting `fire_and_forget` to properly define the necessary framework features for `co_await`, but I think that was just a mistake on my part. I'll have a look again!